xml/Makefile.am

```diff
@@ -106,39 +106,63 @@ pacemaker-%.rng: $(RNG_files) best-match.sh Makefile.am
 	echo "<grammar xmlns='http://relaxng.org/ns/structure/1.0' datatypeLibrary='http://www.w3.org/2001/XMLSchema-datatypes'>" >> $@
 	echo "  <start>" >> $@
 	echo "    <element name='cib'>" >> $@
-	$(top_srcdir)/xml/best-match.sh cib $(*) $(@) "      "
+	./best-match.sh cib $(*) $(@) "      "
```

Comment:
Right, top_srcdir can be relative ("..") so I don't see any benefit in how it was done before. Perhaps ${srcdir}/best-match.sh would be more automake-y but probably not any better.